.circleci/config.yml

```diff
@@ -121,15 +121,16 @@ jobs:
           command: |
             sudo apt-get update && sudo apt-get install -y expect
             pushd $HOME/project/.circleci/resources
+            kubectl create namespace model-engine
```

Comment:
is integration test model-engine actually deployed in this namespace?

Reply:
seems like from this https://github.com/scaleapi/llm-engine/blob/main/.circleci/config.yml#L164 that the helm chart is installed in the default namespace but from here https://github.com/scaleapi/llm-engine/blob/tiffany/fine-tune-e2e/charts/model-engine/values_circleci.yaml#L97 the endpoints are deployed in the model-engine namespace

Reply:
`helm install model-engine model-engine --values model-engine/values_circleci_subst.yaml --set tag=$CIRCLE_SHA1 --atomic --debug` does not specify which namespace to install the chart. it installs a chart defined in `./model-engine` folder, with name `model-engine`

endpoint and model-engine don't need to be in the same namespace

Reply:
can you try to remove these if this is actually not used?

Reply:
actually the namespace `model-engine` is used for endpoints and batch jobs which was why I needed to create a new postgres secret there

Reply:
is this specific for finetune integration test? i don't think this blocks any other integration tests

Reply:
the postgres secret is specific because we're actually checking if the batch job gets created in the fine-tune integration test. there's one batch job integration test but it doesn't actually check for a successful creation which is why we didn't need this secret before.